state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -615,6 +615,7 @@ pub mod pallet {
 			amount: AssetAmount,
 		},
 		CcmFailed {
+			request_id: SwapRequestId,
```

Comment:
Personally would've made this change in a separate PR as it's not related to the error stuff at all

Reply:
Also this issue was in triage, not in TODO so strictly speaking wasn't ready to pick up / hadn't be decided that we should do it (though in this case I do think we should do it).

Reply:
This is removed. WIll be done in a separate PR after it is triaged